gtsam/hybrid/DCFactorGraph.h

```diff
@@ -0,0 +1,33 @@
+/* ----------------------------------------------------------------------------
```

Comment:
Gotcha. MIT's copyright was a simple line saying  "From the ambitious folks at MRG". Since this code will be a part of GTSAM, how would the licensing work there?